components/micro-gateway-core/src/main/ballerina/gateway/executiontime_util.bal

```diff
@@ -58,20 +31,20 @@ function generateExecutionTimeEvent(http:FilterContext context) returns Executio
         executionTimeDTO.keyType = PRODUCTION_KEY_TYPE;
     }
     executionTimeDTO.apiName = getApiName(context);
-    executionTimeDTO.apiVersion = getAPIDetailsFromServiceAnnotation(reflect:getServiceAnnotations(context.serviceType)).
-    apiVersion;
+    executionTimeDTO.apiVersion = getAPIDetailsFromServiceAnnotation(reflect:getServiceAnnotations(context.serviceType))
```

Comment:
Previous version we had the API version as <apiProvider>:v<apiversion> kind of a format (in synapse gateway event). Can you check whether it is the same here. If it is the same, can you remove it and get the version only. (same thing is done in synapse event. this would make both events similar)

Reply:
In here the apiVersion will get values such as 1.0.0 etc. The provider name or prefix "v" is not added.